src/main/java/service/DataProcesorService.java

```diff
@@ -0,0 +1,21 @@
+package service;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface DataProcesorService {
```

Comment:
I would split this service into Transaction/DataParser and Transaction/DataProcessor